archeologie-medievale.csl

```diff
@@ -273,7 +273,7 @@
                   <text macro="publisher"/>
                 </group>
               </else-if>
-              <else-if type="chapter paper-conference" match="any">
+              <else-if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
```

Comment:
That looks wrong because one of the cases before is
```
<else-if type="article-journal article-magazine article-newspaper broadcast personal_communication thesis entry-dictionary entry-encyclopedia" match="any">
```